README.md

```diff
@@ -17,6 +17,25 @@ Using a non-standard approach to store blog posts has some pros and cons. Here's
 - You should iterate through the collection with `for item in site.<collection name>`
 - Drafts are not possibile with collections
 
+### Post Author
+
+You can specify the author's name for every blog post by setting the front-matter property like so:
+
+`author: "Author Name and Surname"`
+
+You can also set the author's avatar for every blog post by setting the front-matter property like so:
+
+`author_avatar: "author_name_and_surname.extension"`
+
+Jekyll will find the avatar image file in `/assets`
+
+Please note that if no `author` is specified, Jekyll will default to "Daniele Polencic".
```

Comment:
LOL! Not sure if I should be flattered or... scared 😂

Reply:
None of the above :) You're just generic :) :)